aim/agent/aid/universes/aci/converter.py

```diff
@@ -994,7 +995,16 @@ def _squash(self, converted_list):
                               enumerate(klass.identity_attributes)])))
             for k, v in res.__dict__.iteritems():
                 if isinstance(v, list):
-                    current.__dict__.setdefault(k, []).extend(v)
+                    for item in v:
+                        if isinstance(item, dict) and '_converter_id' in item:
+                            id_ = (klass.__name__, ) + item.pop(
```

Comment:
id_ should also include k, the AIM resource attribute

Reply:
good catch!